README.md

```diff
@@ -17,46 +17,177 @@ Or directly:
 
 The bitpay client creates a cryptographically secure connection to your server by pairing an API code with keys generated by the library. The client can be initialized with pre-existing keys passed in as a pem file, or paired if initialized with a pem file and a tokens hash. Examples can be found in the cucumber step helpers.
 
-## Basic Usage
+## Command-line Interface (CLI)
```

Comment:
Since this gem is included in the command line gem, any instructions that depend on installing the bitpay-client gem should be in that readme, and definitely not in this readme.